### PR TITLE
Sync scroll

### PR DESCRIPTION
**Pull request by tharshada**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ npm run change`

#### Description of changes
Calling syncScroll after forceUpdate for sticky

#### Focus areas to test

(optional)

### CODE REVIEW

**Comment by tharshada** [comment]
@leddie24 can you please review/approve?

**Comment by KevinTCoughlin** [comment]
@tharshada can you please provide additional context regarding this change? Is the change fixing an existing issue? If so what is that issue?

Providing this information helps reviewers sign-off with confidence which generally reduces time-to-merge of a pull-request.

File: common/changes/office-ui-fabric-react/syncScroll_2018-11-27-13-32.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "sync scroll after sticky forceUpdate",
```
Comment (Vitalius1) [inline common/changes/office-ui-fabric-react/syncScroll_2018-11-27-13-32.json]: Can you elaborate the comment in a more descriptive way what is the change about? This goes to release notes that are read by all consumers and your comment doesn't say them much what they get with a new release. A good way to start would be 'Sticky: do this to fix that...`

Comment (tharshada) [reply common/changes/office-ui-fabric-react/syncScroll_2018-11-27-13-32.json]: 1) If the distanceFromTop is changed, a forceUpdate would happen.
2) The component would be re-rendered but its scrollLeft would not be synced to scrollablepane content container.
[Here is an example.](https://microsoft-my.sharepoint.com/:f:/p/hatrived/EvtojtX22G1EjEhIrZz_sREBLXCZXNK4JKFIqvgqrj0NOQ?e=SiV5H6)

**Comment by tharshada** [comment]
> @tharshada can you please provide additional context regarding this change? Is the change fixing an existing issue? If so what is that issue?
> 
> Providing this information helps reviewers sign-off with confidence which generally reduces time-to-merge of a pull-request.
@KevinTCoughlin , this is a bug fix. 
[Check this ](https://microsoft-my.sharepoint.com/:f:/p/hatrived/EvtojtX22G1EjEhIrZz_sREBLXCZXNK4JKFIqvgqrj0NOQ?e=SiV5H6)

File: common/changes/office-ui-fabric-react/syncScroll_2018-11-27-13-32.json (hunk 1 above)
Comment (Vitalius1) [reply common/changes/office-ui-fabric-react/syncScroll_2018-11-27-13-32.json]: @tharshada thanks for the video demo. It clearly shows there is a bug. Could you please take this 2 points you stated above, combine them and make it as the comment for your changelog?
@leddie24 or @ThomasMichon  can you please review if this bug would be fixed by this PR and if it's not introducing some performance regression. Thanks.

Comment (Vitalius1) [reply common/changes/office-ui-fabric-react/syncScroll_2018-11-27-13-32.json]: Thanks a lot for the change.

**Comment by KevinTCoughlin** [comment]
> 
> 
> > @tharshada can you please provide additional context regarding this change? Is the change fixing an existing issue? If so what is that issue?
> > Providing this information helps reviewers sign-off with confidence which generally reduces time-to-merge of a pull-request.
> > @KevinTCoughlin , this is a bug fix.
> > [Check this ](https://microsoft-my.sharepoint.com/:f:/p/hatrived/EvtojtX22G1EjEhIrZz_sREBLXCZXNK4JKFIqvgqrj0NOQ?e=SiV5H6)

Thanks for clarifying. In the future that is useful information to put in the PR description to help reviewers who may not have full context (and are on shield tasked with closing PRs).

**Comment by KevinTCoughlin** [comment]
Revised changelog message for improved readability in OUFR release notes... https://github.com/OfficeDev/office-ui-fabric-react/pull/7232/commits/6451d629dfebb3bc7c3093c65fd30b3ce13afdc4. Merging after build succeed given the above approval.

**Pull request merged**